lib/rules/value-in-payable.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @fileoverview Ensure "msg.value" is only used in functions with the "payable" modifier
+ * @author Ivan Mushketyk <ivan.mushketik@gmail.com>
+ */
+
+"use strict";
+
+module.exports = {
+
+    meta: {
+
+        docs: {
+            recommended: true,
+            type: "error",
+            description: "Ensure 'msg.value' is only used in functions with the 'payable' modifier"
+        },
+
+        schema: []
+    },
+
+    create: function(context) {
```

Comment:
please change it to `create(context) {`. + I also encourage you to use as many ES6 features as you'd like (I'm anyway moving the old solium code into es6 these days)